### ISSUE TITLE
Dont show any informacion

### ISSUE THREAD

**Issue by Thermaldude** [iptv-org/epg#1751]
### Description

Hello guys, i try put epg in smaters pro app, but in app tell epg find for 0 channels, anyone know how fix that? Im new in this topic, thks for your amazing work 
![Screenshot_20230121_082747_IPTV Smarters Pro](https://user-images.githubusercontent.com/122721926/213848909-58819538-5a68-4721-a069-70a6739be059.jpg)
![Screenshot_20230121_082835_IPTV Smarters Pro](https://user-images.githubusercontent.com/122721926/213848920-c7957368-3b36-4744-8609-15f5681d942a.jpg)

ThKS for your help

**Comment by PopeyeTheSai10r** [comment]
Hello, Can you please confirm the EPG URL you are using?

**Comment by Thermaldude** [comment]
> Hello, Can you please confirm the EPG URL you are using?

Yes, is meo, i try in tivimate and work now, only dont work in iptv smaters pro

**Comment by PopeyeTheSai10r** [comment]
Sounds like a problem with the iptv smaters pro and not the EPG Urls.
